metadata.yaml

```diff
@@ -103,6 +103,8 @@ runtimes:
     name: 8.x (Community)
   - id: redhat
     name: 8.x (RHOAR)
+  - id: v10-redhat
```

Comment:
Is this because the ID for `8.x` was just `redhat` and we need to start distinguishing versions?

Reply:
Right. Actually depends on if you want to keep 8.x boosters available together with 10.x?

E.g. with WildFly Swarm (and Vert.x too, I think), we're just keeping the latest. Spring Boot keeps "current" and "previous".

Reply:
@Ladicek beat me to it. but yes.